feature-libs/customer-ticketing/components/customer-ticketing-list/customer-ticketing-list.component.html

```diff
@@ -1,13 +1,110 @@
-<h2 class="cx-title d-none d-lg-block d-xl-block">
-  <!-- {{ 'title' | cxTranslate }} -->
-</h2>
-
 <ng-container>
-  <div class="cx-header-actions">
-    <!-- <cx-customer-ticketing-list-actions></cx-customer-ticketing-list-actions> -->
-  </div>
-</ng-container>
+  <ng-container *ngIf="tickets | async as tickets">
+    <h3 class="cx-ticketing-list-title">
+      <span class="cx-ticketing-list-title-text">
+        {{ 'customerTicketing.requestTitle' | cxTranslate }}
+      </span>
+      <span class="cx-ticketing-list-title-count">
+        ({{ tickets.length }})
+      </span>
+    </h3>
+    <div class="cx-ticketing-list-subtitle">
+      {{ 'customerTicketing.sortSubtitle' | cxTranslate }}
+    </div>
+    <select
+      class="cx-ticketing-list-select"
```

Comment:
Is cx-ticketing-list-select following spartacus rules?  Why do we need to create a new class?
Same question for:
cx-ticketing-list-title
cx-ticketing-list-title-text
cx-ticketing-list-subtitle

Reply:
I'm just creating this classes for css styling